tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -566,27 +566,25 @@ def zip(datasets):
     For example:
 
     ```python
-    # NOTE: The following examples use `{ ... }` to represent the
-    # contents of a dataset.
-    a = { 1, 2, 3 }
-    b = { 4, 5, 6 }
-    c = { (7, 8), (9, 10), (11, 12) }
-    d = { 13, 14 }
+    a = Dataset.range(1, 4)  # [ 1, 2, 3 ]
+    b = Dataset.range(4, 7)  # [ 4, 5, 6 ]
+    c = Dataset.from_tensor_slices([[7, 8], [9, 10], [11, 12]])
```

Comment:
optional: this could be `Dataset.range(7, 13).batch(2)  # ==> [[7, 8], [9, 10], [11, 12]]`

Reply:
Applied this method in two places (line 571 and 607)